server/test/test_api.py

```diff
@@ -76,21 +76,23 @@ def test_put_layout_fbs(self):
         # attempt to reembed with umap over 100 cells.
         endpoint = "layout/obs"
         url = f"{self.URL_BASE}{endpoint}"
-        header = {"Accept": "application/octet-stream"}
         data = {}
         data["filter"] = {}
         data["filter"]["obs"] = {}
         data["filter"]["obs"]["index"] = list(range(100))
         data["method"] = "umap"
-        result = self.session.put(url, headers=header, json=data)
+        result = self.session.put(url, json=data)
 
         self.assertEqual(result.status_code, HTTPStatus.OK)
-        df = decode_fbs.decode_matrix_FBS(result.content)
-        self.assertEqual(df["n_rows"], 100)
-        self.assertEqual(df["n_cols"], 2)
-        cols = list(df["col_idx"])
-        self.assertTrue(cols[0].startswith("reembed:umap_") and cols[0].endswith("_0"))
-        self.assertTrue(cols[1].startswith("reembed:umap_") and cols[1].endswith("_1"))
+        result_data = result.json()
+        self.assertIsInstance(result_data, dict)
+        self.assertEqual(result_data.get("type", None), "float32")
```

Comment:
Are the defaults for the `get` functions there on purpose? It seems like you never actually want the default value to occur since you assert for equality so the `get` function failing is just wrong behavior right? In which case, I'd remove the default function to assert that result_data.get("type") is always expected to return `float32`. Ditto in lines below.

Also.. the default `default` value is `None` anyways which makes it redundant :) 

Ditto in other locations.

Reply:
I'm just not very pythonic and was doing a lot of cutting/pasting.  Will cleanup!